src/entities/addLiquidityBoosted/doAddLiquidityUnbalancedQuery.ts

```diff
@@ -0,0 +1,31 @@
+import { createPublicClient, http, getAddress } from 'viem';
+
+import { BALANCER_COMPOSITE_LIQUIDITY_ROUTER, CHAINS } from '@/utils';
+
+import { AddLiquidityUnbalancedInput } from '../addLiquidity/types';
+import { Address } from '@/types';
+
+import { balancerCompositeLiquidityRouterAbi } from '@/abi';
+
+export const doAddLiquidityUnbalancedQuery = async (
+    { rpcUrl, chainId, amountsIn }: AddLiquidityUnbalancedInput,
```

Comment:
Same as `doAddLiquidityProportionalQuery`

Reply:
addresses in a larger refactor to use userData and userAddress